DevCycle.SDK.Server.Local/Api/DVCEventsApiClient.cs

```diff
@@ -21,17 +21,16 @@ public DVCEventsApiClient()
         {
         }
 
-        public DVCEventsApiClient(string environmentKey, DVCLocalOptions options = null, RestClientOptions restClientOptions = null)
+        public DVCEventsApiClient(string environmentKey, DVCLocalOptions options = null, DvcRestClientOptions restClientOptions = null)
         {
             options ??= new DVCLocalOptions();
-            restClientOptions ??= new RestClientOptions()
+            DvcRestClientOptions clientOptions = restClientOptions?.Clone() ?? new DvcRestClientOptions()
             {
                 BaseUrl = new Uri(options.EventsApiUri)
             };
             options.EventsApiCustomHeaders ??= new Dictionary<string, string>();
-            if (string.IsNullOrEmpty(restClientOptions.BaseUrl?.ToString()))
-                restClientOptions.BaseUrl = new Uri(options.EventsApiUri);
-            restClient = new RestClient(restClientOptions);
+ 
+            restClient = new RestClient(clientOptions);
```

Comment:
this is where the BaseUrl was incorrectly using the cdn url (because restClientOptions.BaseUrl had been mutated)